xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,249 @@
+      are provided by &suse; via Grafana community dashboards.
+    </para>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-terms">
+    <title>Terminology</title>
+    <para></para>
+    <variablelist>
+      <varlistentry>
+        <term>Grafana</term>
+        <listitem>
+          <para>
+            An interactive, visualization and analytics web application.
+            With dashboards, it provides methods to visualize, explore,
+            alert on, and query your metrics.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Prometheus</term>
+        <listitem>
+          <para>
+            A systems monitoring and alerting toolkit. It collects and evaluates
+            metrics, displays the result, and trigger possible alerts
+            when a observed condition is true.
+            Metrics can be collected from different targets at given intervals.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-hana-db-exp">
+    <title>&hana; Database Exporter</title>
+    <para>
+      Export &hana; database metrics.
+      The exporter is able to export the
+      metrics from more than one database or tenant. This option
+      (<option>multi_tenant</option>) is enabled by default.
+    </para>
+    <formalpara>
+      <title>Package:</title>
+      <para><package>prometheus-hanadb_exporter</package></para>
+    </formalpara>
+    <formalpara>
+      <title>Link:</title>
+      <para>
+        See <link xlink:href="https://github.com/SUSE/hanadb_exporter"/>.
+      </para>
+    </formalpara>
+    <itemizedlist>
+      <title>Export Metrics in Prometheus Format</title>
+      <listitem>
+        <para>
+          Memory metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          CPU metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Disk usage metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          I/O metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Network metrics
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Top queries consuming time and memory
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          System replication metrics
+        </para>
+      </listitem>
+    </itemizedlist>
+    <remark>toms 2020-07-02: Regarding Installation: Do we need really need to
+      describe this via a virtual Python environment or can this be installed
+      through an RPM package?</remark>
+    <!--
+    1. Plain text password
+    2. HANA Keystore
+    3. SAP HANA Driver installation.
+    -->
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-ha-cluster-exp">
+    <title>&ha; Cluster Exporter</title>
+    <para>
+      Enables monitoring of Pacemaker on &ha; clusters.
```

Comment:
This is partially true. The exporter monitors others tools like `corosync`, `sbd`, `drbd`, etc
Anyway, this is commented in the next paragraph, so maybe it's good enough

Reply:
Ok, I've amended the sentence:

    Enables monitoring of Pacemaker, Corosync, SBD, DRBD and others on high availability clusters.

Better?

Reply:
Yes, better